components/Cards/index.js

```diff
@@ -17,3 +17,47 @@
 // </div>
 //
 // Create a card for each of the articles and add the card to the DOM.
+const articleCards = document.querySelector(".cards-container");
+
+axios
+    .get('https://lambda-times-backend.herokuapp.com/articles')
+    .then(response => {
+        console.log(response.data.articles);
+        const mainContainer = Object.entries(response.data.articles);
+
+        mainContainer.forEach(subject => {
+            subject[1].forEach(data => {
+                const newCard = Article(data);
+                articleCards.append(newCard);
+            })
+        })
```

Comment:
Pat, I like this a lot, man. Your syntax is really clean and readable here. The variables here: mainContainer, subject, subject[1], data... etc --||this whole block||-- reads like English to me. And its really not a simple code block. If you can make a chained, double foreach, server requesting, factoryFucntion easy to read/understand -you're doing good things.